exercise-002/main.cpp

```diff
@@ -29,6 +31,38 @@ auto main(int argc, char **argv) -> int
     fmt::print("Hello, {}!\n", app.get_name());
 
     /* INSERT YOUR CODE HERE */
+    app.add_option("-c,--count", count );
+
+    std::vector<int> einVector(count);
+    for (int i = 0; i < count; ++i) 
+    {
+        einVector[i] = rand() % 100 + 1;
+        fmt::print("{}\n", einVector[i]);
+    }
+
+    //sortiert
+    auto start = std::chrono::system_clock::now();
+    std::sort(einVector.begin(), einVector.end());
+    auto end = std::chrono::system_clock::now();
+    //auto elapsed = end - start;
+    auto elapsed = std::chrono::duration_cast<std::chrono::nanoseconds>(end - start);
+
+    printf("nun nach size sortiert: \n");
+    for (int j = 0; j < sizeof(einVector); j++ )
```

Comment:
```suggestion
    for (int j = 0; j < einVector.size(); j++ )
```

This may not work as expected! See https://godbolt.org/z/cWhbEve5W